src/d3a/models/area/markets.py

```diff
@@ -117,6 +119,8 @@ def create_future_markets(self, current_time, is_spot_market, area):
             if timeframe not in markets:
                 # Create markets for missing slots
                 market = market_class(
+                    simulation_id=str(uuid.uuid4()),
```

Comment:
To be replaced with actual `simulation_id`.

Reply:
Why can't it be replaced with the actual simulated id?